BrainCraft_Google_Assistant/gv_buttontotalk.py

```diff
@@ -19,13 +19,14 @@
 import logging
 import os
 import os.path
-import pathlib2 as pathlib
 import sys
 import time
 import uuid
+import pathlib2 as pathlib
```

Comment:
@ladyada I did some of the linting for this project, but there were quite a few messages that I wasn't really sure how to fix.

Reply:
Here is the list of stuff I did not fix:
```
BrainCraft_Google_Assistant/gv_buttontotalk.py 134: Either all return statements in a function should return an expression, or none of them should. (inconsistent-return-statements)
BrainCraft_Google_Assistant/gv_buttontotalk.py 383: Unused argument 'args' (unused-argument)
BrainCraft_Google_Assistant/gv_buttontotalk.py 383: Unused argument 'kwargs' (unused-argument)
BrainCraft_Google_Assistant/gv_buttontotalk.py 590: No value for argument 'api_endpoint' in function call (no-value-for-parameter)
BrainCraft_Google_Assistant/gv_buttontotalk.py 590: No value for argument 'credentials' in function call (no-value-for-parameter)
BrainCraft_Google_Assistant/gv_buttontotalk.py 590: No value for argument 'project_id' in function call (no-value-for-parameter)
BrainCraft_Google_Assistant/gv_buttontotalk.py 590: No value for argument 'device_model_id' in function call (no-value-for-parameter)
BrainCraft_Google_Assistant/gv_buttontotalk.py 590: No value for argument 'device_id' in function call (no-value-for-parameter)
BrainCraft_Google_Assistant/gv_buttontotalk.py 590: No value for argument 'device_config' in function call (no-value-for-parameter)
BrainCraft_Google_Assistant/gv_buttontotalk.py 590: No value for argument 'lang' in function call (no-value-for-parameter)
BrainCraft_Google_Assistant/gv_buttontotalk.py 590: No value for argument 'display' in function call (no-value-for-parameter)
BrainCraft_Google_Assistant/gv_buttontotalk.py 590: No value for argument 'verbose' in function call (no-value-for-parameter)
BrainCraft_Google_Assistant/gv_buttontotalk.py 590: No value for argument 'input_audio_file' in function call (no-value-for-parameter)
BrainCraft_Google_Assistant/gv_buttontotalk.py 590: No value for argument 'output_audio_file' in function call (no-value-for-parameter)
BrainCraft_Google_Assistant/gv_buttontotalk.py 590: No value for argument 'audio_sample_rate' in function call (no-value-for-parameter)
BrainCraft_Google_Assistant/gv_buttontotalk.py 590: No value for argument 'audio_sample_width' in function call (no-value-for-parameter)
BrainCraft_Google_Assistant/gv_buttontotalk.py 590: No value for argument 'audio_iter_size' in function call (no-value-for-parameter)
BrainCraft_Google_Assistant/gv_buttontotalk.py 590: No value for argument 'audio_block_size' in function call (no-value-for-parameter)
BrainCraft_Google_Assistant/gv_buttontotalk.py 590: No value for argument 'audio_flush_size' in function call (no-value-for-parameter)
BrainCraft_Google_Assistant/gv_buttontotalk.py 590: No value for argument 'grpc_deadline' in function call (no-value-for-parameter)
BrainCraft_Google_Assistant/gv_buttontotalk.py 590: No value for argument 'once' in function call (no-value-for-parameter)
```

Reply:
yeah skip that, its not our code